pkg/antctl/raw/multicluster/init.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+)
+
+type initOptions struct {
+	namespace  string
+	clusterSet string
+	clusterID  string
+	output     string
+}
+
+var initOpts *initOptions
+
+func (o *initOptions) validate() error {
+	if o.namespace == "" {
+		return fmt.Errorf("Namespace is required")
+	}
+	if o.clusterSet == "" {
+		return fmt.Errorf("ClusterSet is required")
+	}
+	if o.clusterID == "" {
+		return fmt.Errorf("ClusterID is required")
+	}
+	return nil
+}
+
+var initExample = strings.Trim(`
+# Initialize ClusterSet in the given Namespace of the leader cluster.
+  $ antctl mc init --namespace antrea-multicluster --clusterset test-clusterset --clusterid test-cluster-north
+`, "\n")
+
+func NewInitCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "init",
+		Short:   "Initialize ClusterSet in the given Namespace of the leader cluster",
+		Args:    cobra.MaximumNArgs(0),
+		Example: initExample,
+		RunE:    initRunE,
+	}
+
+	o := initOptions{}
+	initOpts = &o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Antrea Multi-cluster Namespace")
+	command.Flags().StringVarP(&o.clusterSet, "clusterset", "", "", "ClusterSet ID of the leader cluster")
+	command.Flags().StringVarP(&o.clusterID, "clusterid", "", "", "ClusterID of the leader cluster")
+	command.Flags().StringVarP(&o.output, "output", "o", "", "Output config file for the member cluster")
+
+	return command
+}
+
+func initRunE(cmd *cobra.Command, args []string) error {
+	if err := initOpts.validate(); err != nil {
+		return err
+	}
+	k8sClient, err := common.NewClient(cmd)
+	if err != nil {
+		return err
+	}
+	createdRes := []map[string]interface{}{}
+	var createErr error
+	defer func() {
+		if createErr != nil {
+			fmt.Fprintf(cmd.OutOrStderr(), "Failed to init the Antrea Multi-cluster, Deleting the created resources.\n")
```

Comment:
", Deleting" -> ". Deleting"

Maybe remove "." at the end to be consistent with other messages.

Reply:
Removed "." and other two "."